### ISSUE TITLE
worse result on higher checkpoint than the lower checkpoint

### ISSUE THREAD

**Issue by ghost** [Gal4way/TPD#6]
Hi, I am training it on my dataset, producing worse result on higher checkpoint than the lower checkpoint.
For example, results on ckpnt 50 is worse than that of ckpnt 35.

Any idea why?

Is it could be because I am not training it in one go, let it train for 10 epochs then stop and then start again from last checkpoint?
I think it will start from the initial learning rate mentioned in the config file then, right?

linear_start: 0.00085
linear_end: 0.0120

**Comment by xdfeng07** [comment]
Hi, have you tried your model of different epochs on test datasets and see the visual quality of the tryon result images?

**Comment by Gal4way** [comment]
@elenakovacic 
Is it could be because I am not training it in one go, let it train for 10 epochs then stop and then start again from last checkpoint?
I think it will start from the initial learning rate mentioned in the config file then, right?


Yes. If the training process is not completed in one go and you resume training from the initial learning rate, it may lead to suboptimal performance. To address this issue, when you want to continue training an existing checkpoint, it's recommended to set the f_start parameter to 1. By doing so, the learning rate will not be reset, allowing the model to seamlessly continue learning from where it left off.

![image](https://github.com/Gal4way/TPD/assets/70054256/79e34c1c-b39c-4bd0-bad5-19c7349038ca)
